qa/common/nightly_email_helper.py

```diff
@@ -39,7 +39,7 @@
 
```

Comment:
'base64' is not used. You should be able to remove that import

Reply:
Removed.